poi/poi_types.xml

```diff
@@ -33,14 +33,27 @@
 	<poi_additional name="roundtrip_no" tag="roundtrip" value="no"/>
 
 	<!-- should be not a text though it's not implemented without listing all possible values -->
+
+	<!-- OBF-GPX point -->
 	<poi_additional name="gpx_icon" tag="gpx_icon" type="text"/>
 	<poi_additional name="gpx_bg" tag="gpx_bg"  type="text"/>
 	<poi_additional name="color" tag="colour" type="text"/>
-<!--
-	<poi_additional name="color_int" tag="color_int" type="text" lang="true" order="82"/>
--->
 
-	
+	<!-- OBF-GPX track extensions (color=gpx_color, width already defined, point_groups todo -->
+	<poi_additional name="show_arrows" tag="show_arrows" type="text"/>
```

Comment:
perhaps better to group to poi_additional_category

Reply:
reorganized with gpx_ prefix